_posts/2020-09-28-release-statefun-2.2.0.md

```diff
@@ -0,0 +1,179 @@
+  was reduced to improve stability and resource consumption, in the face of a slow-responding remote function.
+
+### Operational observability of a StateFun Application
+
+One major goal of this release was to take a necessary step towards supporting auto-scaling of remote functions. Towards that end,
+we've exposed several metrics related to workload of remote functions and resulting backpressure applied by the function
+dispatchers. This includes the following:
+
+* Per function type invocation duration / latency histograms
+* Per function type backlog size
+* Per JVM (StateFun worker) and per function type number of in-flight invocations
+
+The full list of metrics and their descriptions can be found [here](https://ci.apache.org/projects/flink/flink-statefun-docs-release-2.2/deployment-and-operations/metrics.html).
+
+### Fine-grained control over remote connection lifecycle
+
+With this release it is possible to set individual timeouts for overall duration and individual read and write IO operations
+of HTTP requests with remote functions. You can find the corresponding field names in a function spec that defines
+these timeout values [here](https://ci.apache.org/projects/flink/flink-statefun-docs-release-2.2/sdk/index.html#defining-functions).
+
+## 3rd Party SDKs
+
+Since the last release, we've seen new 3rd party SDKs for different languages being implemented on top of StateFun's
+remote function HTTP request-reply protocol, including a [Go]() and [Rust]() implementation. While these SDKs are not
+endorsed or maintained by the Apache Flink PMC and currently not part of the current releases, it is great to see these
+new additions that demonstrates extensibility of the framework.
+
+For that reason, we've added
+a new [page in the documentation](https://ci.apache.org/projects/flink/flink-statefun-docs-release-2.2/sdk/external.html)
+to list the 3rd party SDKs that the community is aware of. If you've also worked on a new language SDK for StateFun that
+is stable and am planning to continue maintaining it, please consider letting the community know of your work by
+submitting a pull request to add your project to the list!
+
+## Important Patch Notes
+
+Below is a list of user-facing interface and configuration changes, dependency version upgrades, or removal of supported versions that would be
+important to be aware of when upgrading your StateFun applications to this version:
+
+* [[FLINK-18812](https://issues.apache.org/jira/browse/FLINK-18812)] The Flink version in StateFun 2.2 has been upgraded to 1.11.1.
+* [[FLINK-19203](https://issues.apache.org/jira/browse/FLINK-19203)] Upgrade Scala version to 2.12, and dropped support for 2.11.
+* [[FLINK-19190](https://issues.apache.org/jira/browse/FLINK-19190)] All existing metric names have been renamed to be camel-cased instead of snake-case, to conform with Flink metric naming conventions. This breaks existing deployments if you depended on previous metrics.
```

Comment:
```suggestion
* [[FLINK-19190](https://issues.apache.org/jira/browse/FLINK-19190)] All existing metric names have been renamed to be camel-cased instead of snake-cased, to conform with the Flink metric naming conventions. **This breaks existing deployments if you depended on previous metrics**.
```